src/templates/caseStudy.js

```diff
@@ -36,11 +36,13 @@ const CaseStudy = ({
       </Grid>
       <GreyBackgroundOffset topMargin topOffset={-150}>
         <Grid>
-          <VideoSection src={videoInfo[0]} />
           <Padding
             top={{ smallPhone: 3, tablet: 4 }}
-            bottom={{ smallPhone: 3.5, tablet: 4 }}
+            bottom={{ smallPhone: 3, tablet: 4 }}
           >
+            <VideoSection src={videoInfo[0]} />
+          </Padding>
```

Comment:
We can do a lot of improvements to this section here. Specifically the whole HTML structure looks a bit suspect here.

Reply:
Moving the `VideoSection` into the `Padding` removes the `Grid > Row > Col`  markup we want. I would opt for putting the top padding on the VideoSection and the bottom padding on the  `SeconTextSection`. 

To be honest I'd even argue this components (FirstTextSection, SecondTextSection) don't even need to be broken out into individual components. This separation isn't really necessary and just means that the HTML markup we want with the grid system is obscured.

Reply:
@omonk you're right, we definitely should avoid breaking the `Grid > Row > Col` structure. I didn't notice VideoSection was an actual Row container. PR updated to remove this `<Padding>` components from between grid sub-component layers ;)